### ISSUE TITLE
FRT isn't saved for non-silent requests

### ISSUE THREAD

**Issue by imakhalova** [AzureAD/azure-activedirectory-library-for-android#671]
Scenario: acquireToken -> input password -> serialize -> got exception - no FRT found

**Comment by weijjia** [comment]
Can you give more details? What do you mean by serialize here?

**Comment by imakhalova** [comment]
AuthenticationContext.serialize(final String uniqueUserId) throws AuthenticationException

**Comment by heidijinxujia** [comment]
Hi @imakhalova , I am trying to repro this issue. But I got the FRT token serialized successfully. I tried with the first party app and has the FRT stored in the local cache, that is the family client ID if the token cache item is not null. Called serialize() and got the serialized string successfully. Which version of adal are you using? Have you checked the cache if the FRT is return by the server before trying serialization?
